static/app/utils/theme.tsx

```diff
@@ -677,6 +677,31 @@ const commonTheme = {
     lineHeightBody: '1.4',
   },
 
+  /**
+   * Common styles for form inputs & buttons, separated by size.
+   * Should be used to ensure consistent sizing among form elements.
+   */
+  form: {
+    default: {
+      height: '40px',
```

Comment:
Explicitly setting a height here to fix the uneven sizing issue we've seen in a few places.

Before:
<img width="315" alt="Screen Shot 2022-01-18 at 4 42 58 PM" src="https://user-images.githubusercontent.com/44172267/150042019-59594372-e625-4658-acd4-d1ed5672e7d7.png">

After:
<img width="315" alt="Screen Shot 2022-01-18 at 4 43 16 PM" src="https://user-images.githubusercontent.com/44172267/150042055-7fc8ad18-3187-4ba7-9492-5bfe2d2b51f7.png">

Reply:
this issue was on my list for a while now... Thanks for fixing it @vuluongj20 👏